src/main.js

```diff
@@ -164,10 +166,16 @@ module.exports = {
               }
             }
             let range
+            const msgLine = line - 1
+            const msgCol = column ? column - 1 : column
             try {
-              range = Helpers.rangeFromLineNumber(
-                textEditor, line - 1, column ? column - 1 : column
-              )
+              if (endColumn && endLine) {
+                validatePoint(textEditor, msgLine, msgCol)
+                validatePoint(textEditor, endLine - 1, endColumn - 1)
+                range = [[msgLine, msgCol], [endLine - 1, endColumn - 1]]
```

Comment:
I assumed `endColumn` is 1-based like all the other points we get from ESLint... did they really make 1/4 of the points 0-based?!?